src/main/java/twilightforest/entity/EntityTFBlockGoblin.java

```diff
@@ -24,22 +26,31 @@
 import net.minecraft.util.EnumHand;
 import net.minecraft.util.ResourceLocation;
 import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.MathHelper;
 import net.minecraft.util.math.Vec3d;
 import net.minecraft.world.World;
 import twilightforest.TFSounds;
 import twilightforest.TwilightForestMod;
 
 import java.util.List;
+import java.util.UUID;
 
 public class EntityTFBlockGoblin extends EntityMob implements IEntityMultiPart {
+	private static final UUID MODIFIER_UUID = UUID.fromString("5CD17E52-A79A-43D3-A529-90FDE04B181E");
+	private static final AttributeModifier MODIFIER = (new AttributeModifier(MODIFIER_UUID, "Drinking speed penalty", -0.25D, 0)).setSaved(false);
```

Comment:
`Drinking speed penalty` wouldn't accurately describe the function of the attribute modifier, I recommend renaming to just `speedPenalty` or something like that.